### PR TITLE
Update depinfo

### PR DESCRIPTION
**Pull request by Midnighter**
* [ ] fix #(issue number)
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry to the [next release](../release-notes/next-release.md)

Update the version of depinfo to >2 and change issue templates to use the form input.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/02-question.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Question
```
Comment (cdiener) [inline .github/ISSUE_TEMPLATE/02-question.yaml]: Shouldn't we rather encourage that questions all go to discussions? Those are much more visible there, especially after they have been answered.

File: .github/ISSUE_TEMPLATE/01-bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+name: 🐞 Bug report
```
Comment (cdiener) [inline .github/ISSUE_TEMPLATE/01-bug-report.yaml:1]: I will assume you have tested all of those and they show up well.

**Review by cdiener — COMMENTED** [review]
Looks good to me, but I would prefer questions in the discussion section I think to have them all in one place.

File: .github/ISSUE_TEMPLATE/01-bug-report.yaml (hunk 2 above)
Comment (Midnighter) [reply .github/ISSUE_TEMPLATE/01-bug-report.yaml:1]: Yes, you can try them out here if you like https://github.com/taxprofiler/taxpasta/issues/new/choose

File: .github/ISSUE_TEMPLATE/02-question.yaml (hunk 1 above)
Comment (Midnighter) [reply .github/ISSUE_TEMPLATE/02-question.yaml]: You're perfectly right, no need to duplicate.

**Comment by Midnighter** [comment]
Is this more like you were thinking?

**Review by cdiener — APPROVED** [review]
Yep, thanks!

**Pull request merged**